changelog/deprecate_class_allocators.dd

```diff
@@ -0,0 +1,20 @@
+Class allocators and deallocators have been deprecated
+
+Class allocators and deallocators have been planned for deprecation for years.  Starting with this release the following code will emit deprecation messages.
+
+---
+class Foo
+{
+    new(size_t size)         // deprecation message
+    {
+        return malloc(size);
+    }
+
+    delete(void* obj)        // deprecation message
+    {
+        free(obj);
+    }
+}
+---
+
```

Comment:
The new `@disable new();` syntax should be mentioned here.

Reply:
Done.